portfolio/src/main/webapp/commentScript.js

```diff
@@ -15,25 +15,42 @@
 document.addEventListener('DOMContentLoaded', displayComments(), false);
 
 function displayComments() {
-  fetch('/comments').then(response => response.json()).then(comments => {
-    const commentsContainer = document.getElementById('comments-container');
-    var html = '<table id="commentsTable" width="80%">';
+  fetch('/comments').then(response => response.json()).then(response => {
+    const commentsContainer = document.getElementById('content');
+    var html = '';
+    if (!response.loggedIn) {
```

Comment:
Is it intended that user can't see any comment when not logged in?

Reply:
Yes, it is intentional.